compiler/src/transform/filter-by-availability.ts

```diff
@@ -267,6 +268,6 @@ const serverless = !!argv.serverless
 // output, if not provided default to schema-filtered.json
 filterSchema(inputPath, outputPath, stack, serverless)
   .catch(reason => {
-    console.error(reason.message? reason.message:reason)
+    console.error((reason.message !== null) ? reason.message : reason)
```

Comment:
Same here with the stricter type checks.

Reply:
Fixed as well in a later commit.